mne_bids/tests/test_mne_bids.py

```diff
@@ -176,6 +178,15 @@ def test_edf():
     cmd = ['bids-validator', '--bep006', output_path]
     run_subprocess(cmd, shell=shell)
 
+    # ensure there is an EMG channel in the channels.tsv:
+    channels_tsv = make_bids_filename(
+        subject=subject_id, session=session_id, task=task, run=run,
+        suffix='channels.tsv', acquisition=acq,
+        prefix=op.join(output_path, 'sub-01/ses-01/eeg'))
+    if op.exists(channels_tsv):
+        df = pd.read_csv(channels_tsv, sep='\t')
```

Comment:
why do you have the `op.exists` if clause? Wouldn't the channels.tsv always exist in this test case?